app/scripts/metamask-controller.js

```diff
@@ -299,6 +299,10 @@ import { snapKeyringBuilder, getAccountsBySnapId } from './lib/snap-keyring';
 import { encryptorFactory } from './lib/encryptor-factory';
 import { addDappTransaction, addTransaction } from './lib/transaction/util';
 import { LatticeKeyringOffscreen } from './lib/offscreen-bridge/lattice-offscreen-keyring';
+///: BEGIN:ONLY_INCLUDE_IF(notifications)
+import { PREINSTALLED_SNAPS } from './snaps/preinstalled-snaps';
```

Comment:
Technically our notifications feature if also tied to preinstalled snaps.

Lets move the preinstalled snaps to the correct code fence.